test.js

```diff
@@ -0,0 +1,62 @@
+var tests = {
+    	/*
+	Given an integer array of size N, compute the sum of all even numbers in this array
+	*/
+	firstTest: function () {
+    		var array = [1, 10, 2, 9, 3, 8, 4, 7, 5, 6];
+	    	var result = 0;
+	    
+		for (var i = 0; i < array.length; i++) {
+			if (array[i]%2 == 0) {
+	  			result += array[i];
+		  	}
+		}
+	    	return result;
+	},
+
+    	/*
+	Given an integer array of size N, identify the largest 2 numbers in this array. Do not sort the array.
+	*/
+    	secondTest: function () { 
+    		var array = [1, 10, 2, 9, 3, 8, 4, 7, 5, 6, 11, 4];
```

Comment:
If I give you a input array of: `[-1, -3, -4]` you will return `0 & 0` a incorrect answer